### PR TITLE
Cdorrian

### PR DESCRIPTION
**Pull request by ConorDorrian1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ M.Sc. DevOps Team 1
   - [Risk Register](#risk-register)
   - [Tenants of Design](#tenants-of-design)
 
-## Introduction
+## Introduction!
```
Comment (ghost) [inline README.md:17]: this is typo here?

File: README.md [hunk 2]
```diff
@@ -83,3 +83,12 @@ Please see our list of Risks
   - branches used
   - version/release management
 - Agile project management methods/principles (jira)
+
+## Consideration
```
Comment (cjfarrelly) [inline README.md]: Needs to go onto the table of contents

**Review by cjfarrelly — CHANGES_REQUESTED** [review]
update the toc

**Review by cjfarrelly — APPROVED** [review]
good to go

**Review by ghost — APPROVED** [review]
looks good

**Pull request merged**